libpod/container_internal_linux.go

```diff
@@ -1779,7 +1797,12 @@ func (c *Container) getHosts() string {
 	if c.Hostname() != "" {
 		if c.config.NetMode.IsSlirp4netns() {
 			// When using slirp4netns, the interface gets a static IP
-			hosts += fmt.Sprintf("# used by slirp4netns\n%s\t%s %s\n", slirp4netnsIP, c.Hostname(), c.config.Name)
+			slirp4netnsIP, err := GetSlirp4netnsGateway(c.slirp4netnsSubnet)
+			if err != nil {
+				logrus.Error("failed to determine slirp4netnsIP: ", err.Error())
```

Comment:
This should be a warning not an error,  If the user can not do anything about it, then it should just warn.

Reply:
Fixed: ef6027b2db9d7ebd4a294af002c08c0c93d0e5db